internal/api/api.go

```diff
@@ -207,9 +209,9 @@ func getS3ServiceClient(ctx context.Context, settings *config.Settings, logger z
 		awsconfig.WithEndpointResolverWithOptions(aws.EndpointResolverWithOptionsFunc(
 			func(service, region string, options ...interface{}) (aws.Endpoint, error) {
 
-				if settings.Environment == "local" {
-					return aws.Endpoint{PartitionID: "aws", URL: settings.CandumpsAWSEndpoint, SigningRegion: settings.AWSRegion}, nil // The SigningRegion key was what's was missing! D'oh.
-				}
+				//if settings.Environment == "local" {
```

Comment:
should this be commented out? or was it just to test locally without s3 docker image running? Could instead check for the awsendpoint containing localhost etc.